packages/shared-ui/src/components/canvas/components/node/node-component/run-button/RunButton.tsx

```
@@ -22,7 +22,7 @@
  */
 
 import React, { ReactElement, useMemo, useState, useEffect, useRef, useCallback } from 'react';
-import { BxPlay, BxStop, BxRefresh } from '../../../../../../components/BoxIcon';
+import { BxPlay, BxStop, BxRefresh } from 'shell';
```

ESLint

[error] 'shell' import is restricted from being used. Renamed: import from 'shell'.

(no-restricted-imports)